src/types/AdsConsent.interface.ts

```diff
@@ -0,0 +1,365 @@
+/**
+ * Under the Google [EU User Consent Policy](https://www.google.com/about/company/consentstaging.html), you must make certain disclosures to your users in the European Economic Area (EEA)
+ * and obtain their consent to use cookies or other local storage, where legally required, and to use personal data
+ * (such as AdID) to serve ads. This policy reflects the requirements of the EU ePrivacy Directive and the
+ * General Data Protection Regulation (GDPR).
+ *
+ * It is recommended that you determine the status of a user's consent at every app launch. The user consent status is held
+ * on the device until a condition changes which requires the user to consent again, such as a change in publishers.
+ *
+ * For more information, see [here](https://developers.google.com/admob/android/eu-consent#delay_app_measurement_optional).
+ */
+
+/**
+ * A AdProvider interface returned from `AdsConsent.getProviders`.
+ */
+export interface AdProvider {
+  /**
+   * A provider company ID.
+   */
+  companyId: string;
+
+  /**
+   * A provider company name.
+   */
+  companyName: string;
+
+  /**
+   * A fully formed URL for the privacy policy of the provider.
+   */
+  privacyPolicyUrl: string;
+}
+
+/**
+ * AdsConsentStatus interface.
+ */
+export interface AdsConsentStatus {
```

Comment:
weren't these also an enum? the interface and enum feels somehow duplicate to me - are both required?